deployment/deployment_test.go

```diff
@@ -326,3 +326,65 @@ func TestUpdateError(t *testing.T) {
 	err := Update(id, role, deploymentConfig, api, buf)
 	assert.EqualError(t, err, "API error (500): Internal Server Error")
 }
+
+func TestAirflowUpgrade(t *testing.T) {
```

Comment:
Can you add the error tests for both of the tests in this file?

Reply:
fixed.